lib/usd/ui/layerEditor/layerEditorWidget.cpp

```diff
@@ -74,12 +78,26 @@ void setupDefaultMenu(SessionState* in_sessionState, QMainWindow* in_parent)
         QObject::connect(createMenu, &QMenu::aboutToShow, in_parent, aboutToShowCallback);
 
         auto optionMenu = menuBar->addMenu(StringResources::getAsQString(StringResources::kOption));
-        auto action = optionMenu->addAction(
+
+        auto autoHideAction = optionMenu->addAction(
```

Comment:
Renamed var, as I added another action below for the menu item to show the layer contents.